reference/types/string.md

```diff
@@ -1,13 +1,25 @@
 # String
 
+## The Concept
+
 A string is a sequence of [characters][type-char] (letters, digits, punctuation, etc.).
 
-# Teaching tactics
+## What to cover
+
+Tell a student what a string is in your language, how strings can be manipulated, and ensure they understand where to look for docs on strings.
+
+- **Explain what strings are in your language?** Is it a sequence of bytes, UTF-8 characters or something else? Are strings objects or primitives?
+- **How are strings brought together?** Do you join strings, add strings, concatenate strings, interpolate strings? It's worth giving a hint as to which is normal, or letting the user know all are ok!
+- **How are strings split apart?** How do I extract a bit of a string?
+- **Where do helper methods for strings live?** If I want to trim a string, or check it's length, how do I do that in your language? Are these string methods or functions that act on strings?
```

Comment:
Asking myself where this lives in an exercise. In the `introduction`? In the `hints` -- which not everyone looks at. Or in the `after.md`?

Reply:
- "Explain what strings are in your language": I'd put that in the `introduction.md` file. 
- "How are strings brought together?": this is where I think my opinion might differ from others. Personally, I'd put this in the `hints.md` file, but I could see an argument made for this being included in `introduction.md`.
- "How are strings split apart?": belongs to `hints.md` I feel.
"Where do helper methods for strings live?": this is where I think my opinion might differ from others. Personally, I'd put this in the `hints.md` file, but I could see an argument made for this being included in `introduction.md`.

Reply:
> How are strings brought together?
> How are strings split apart?"

For clarity, I'm hoping that solving the exercise teaches this, rather than it being something explained in the README, so I agree with putting the explanation in hints.md although linking to the docs in introduction.md is a nice thing to do I think.